hivemq-edge/src/frontend/src/extensions/datahub/components/helpers/CodeEditor.spec.cy.tsx

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
```

Comment:
question: does TypeScript has troubles identifying that this file format / fine name format is Cypress by default?

Reply:
IntelliJ had occasional problem with getting the code completion working. That was the textbook solution to the problem. Became parts of my templates in WebStorm